WordPress/src/main/res/values/strings.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version='1.0' encoding='utf-8'?>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
lol weird that this needed to change 😄

Reply:
This thing has been driving me crazy for the last few releases. The code freeze script says this file has changed, but when I run `git status` from a different terminal, it says there is nothing to commit. Then I continue the script and it fails because the status is not clean. 😠 After the script failure, I am left with this change. This time around I committed the change to see if it helps with future code freezes 🤷‍♂️